config_files/osquery.conf

```diff
@@ -24,10 +24,6 @@
       "snapshot": true,
       "interval": 300
     },
-    "shell_history": {
```

Comment:
There may be customers that actually want and use this data as it stands today. If those customers re-run the script after this change (e.g. a Chef automated deploy in their CI), they would be surprised to see their data disappear.  So as minor as it is, this may need some small comms with the Field team to give them a heads-up and manually add bash_history for customers that need it after this change.